src/coprocessor/endpoint.rs

```diff
@@ -468,17 +485,25 @@ impl<E: Engine> Endpoint<E> {
         req: coppb::Request,
         peer: Option<String>,
     ) -> impl Future<Output = MemoryTraceGuard<coppb::Response>> {
+        let tracker = GLOBAL_TRACKERS.insert(::tracker::Tracker::new(RequestInfo::new(
```

Comment:
why not alloc a global tracker pool, although the pool always occupy allocated memory which is not large, it will not alloc memory for every request.